packages/snaps-cli/src/utils/misc.ts

```diff
@@ -43,14 +43,38 @@ export function setSnapGlobals(argv: Arguments) {
   }
 
   if (Object.prototype.hasOwnProperty.call(argv, 'verboseErrors')) {
-    global.snaps.verboseErrors = Boolean(argv.verboseErrors);
+    global.snaps.verboseErrors = booleanStringToBoolean(argv.verboseErrors);
```

Comment:
Doesn't yargs convert to boolean anyways?

Reply:
Hm, you're right. However, we call this in a middleware that runs before yargs's validation, but after parsing. I'm not sure if yargs will error during parsing or validation if a boolean argument is not the string "true" or "false". If it does error during parsing, then the "bug" exists only in our tests. I should probably verify that before we merge this PR. :thinking:

Reply:
Other than that, this code looks good!